routes/claim/router.js

```diff
@@ -56,8 +56,6 @@ router.post('/new', function(req, res) {
         res.status(500).json({ error : 'database error while finding user' }); // TODO: TEST
       } else if (!user) {
         res.status(404).json({ error : 'user not found' });  // TODO: TEST
-      } else if (user.freelancer) {
-        res.status(452).json({ error : 'user id already claiming' }); // TESTED
```

Comment:
Why is this removed? A single user cannot submit multiple claim requests for the same user, it doesn't make sense.
Is it done client side?